pkgs/development/tools/rome/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, stdenv
+, Security
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rome";
+  version = "11.0.0";
+
+  src = fetchFromGitHub {
+    owner = "rome";
+    repo = "tools";
+    rev = "cli/v${version}";
+    sha256 = "sha256-pQz1hs4oYPPQ8zqkwstDH3VefaYQXJ9uCkeC+0IMeSs=";
+  };
+
+  cargoSha256 = "sha256-9WTjb8pgk/qyIU8mPT6RF2FZptNylvILHMYvZDyRSMs=";
+
+  cargoBuildFlags = [ "--package" "rome_cli"];
+
+  preBuild = ''
+    export RUSTFLAGS="-C strip=symbols"
+    export ROME_VERSION=${version}
```

Comment:
You can set `RUSTFLAGS` & `ROME_VERSION` directly inside the attributes for `buildRustPackage`.